paymentsheet/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignup.kt

```diff
@@ -66,6 +79,7 @@ import kotlinx.coroutines.launch
 import com.stripe.android.uicore.R as StripeUiCoreR
 
 internal const val ProgressIndicatorTestTag = "CircularProgressIndicator"
+private const val LINK_LOGO_SCALE = 0.9f
```

Comment:
This helps aligning text given the Link logo is a bit larger than the Link text on the image.